src/app.py

```
@@ -0,0 +1,87 @@
+        print(f"Error generating question: {e}")
+        sys.exit(1)
+
+def explain_answer(user_answer, correct_answer, question):
+    """Get AI explanation for the answer"""
+    try:
+        response = openai.chat.completions.create(
+            model=MODEL_NAME,
+            messages=[
+                {
+                    "role": "system",
+                    "content": "Explain whether the answer is correct and why. Be encouraging and helpful."
```

Pylint (3.3.7)

[convention] Line too long (107/100)

(C0301)